src/services/services.ts

```diff
@@ -735,6 +735,7 @@ class SymbolObject implements Symbol {
         if (context) {
             if (isGetAccessor(context)) {
                 if (!this.contextualGetAccessorDocumentationComment) {
+                    this.contextualGetAccessorDocumentationComment = emptyArray;

```

Comment:
this was the strategy used before https://github.com/microsoft/TypeScript/pull/46801/files#diff-2dc319893c8723737436e91a4ec39b69623ee0bbeebf0b260778015d79d97632L346 , this was removed there, likely because it looked weird and there was no test proving its usefulness

the same strategy is still used in other places, like here:
https://github.com/microsoft/TypeScript/blob/88809467e8761e71483e2f4948ef411d8e447188/src/services/services.ts#L756-L763

Reply:
Can you add comments so we don't remove it again?

Reply:
done